site/docs/v1/tech/core-concepts/roles.adoc

```diff
@@ -37,19 +37,79 @@ include::docs/v1/tech/core-concepts/_role_attributes.adoc[]
 
 === FusionAuth Application Roles
 
-The FusionAuth application which provides the administrative user interface for the FusionAuth instance provides a number of built-in roles. These can be assigned to any user registered with the FusionAuth admin application. These roles control access to functionality within the FusionAuth administrative user interface. 
+FusionAuth provides an administrative user interface for the running instance with several built-in roles. These can be assigned to any user registered with the FusionAuth admin application. These roles control access to functionality within the FusionAuth administrative user interface.
+
+* <<Role Schema and Exceptions>>
+* <<Application Roles>>
+* <<Special Role - user_support_manager>>
 
 [NOTE]
 ====
 These roles are used only internally to manage authorization within the FusionAuth administrative user interface application.
 
-These roles are *not global* and are not present in any other applications for which FusionAuth provides authentication, authorization or user management.
+These roles are *not global* and are not present in any other applications for which FusionAuth provides authentication, authorization, or user management.
 ====
 
 Below you can see the administrative user interface screen where you assign roles in the FusionAuth application to a user.
 
 image::core-concepts/fusionauth-roles.png[FusionAuth application roles,width=1200]
 
-In general, any role ending in `_viewer` can view entities of a particular type, any role ending in `_manager` can add or edit the entities, and any role ending in `_deleter` can delete entities.
+== Role Schema and Exceptions
+
+The below table outlines how roles were designed at an abstract level. Of course, risk is always relative to the information and organization; even a low-risk role can do significant damage in the wrong hands.
+
+=== Suffix Meaning
+.FusionAuth roles suffix meanings
+[cols="1,1,2"]
+|===
+| Suffix | Relative "Risk" | Meaning
```

Comment:
I like the 'risk' idea, but why don't we say `access level`. I feel like using `"risk"` is unclear.

Reply:
if you want to stick with 'risk', please remove the scare quotes :)